tests/tests.cabal

```diff
@@ -583,9 +613,6 @@ executable errors
                     , BadDataConType
                     , BadDataDeclTyVars
                     , BadGADT
-                    -- , BadPragma0
-                    -- , BadPragma1
-                    -- , BadPragma2
```

Comment:
There are various disabled tests that are removed in this file. Are they beyond any hope of enabling them again?